src/Esfa.Vacancy.Domain/Interfaces/ICacheService.cs

```diff
@@ -5,6 +5,6 @@ namespace Esfa.Vacancy.Domain.Interfaces
 {
     public interface ICacheService
     {
-        Task<T> CacheAsideAsync<T>(string key, Func<Task<T>> actionAsync, TimeSpan timeSpan);
+        Task<T> CacheAsideAsync<T>(string key, Func<Task<T>> actionAsync, TimeSpan? timeSpan = null);
```

Comment:
I think since we are not using timeSpan parameter, we should get rid of it.

Reply:
I left it in to allow any future cache services to override default/single duration